src/mod/injector.rb

```diff
@@ -59,15 +59,15 @@ def inject(gemfile_path, lockfile_path)
 end
 
 class << self
-  def call
-    LOG.info { 'injector:call' }
+  def call(context)
+    LOG.info { "injector:call context:#{context}" }
 
     # TODO: check if nested injection (maybe very early too)
     # TODO: check if injection already performed
 
     # TODO: extract to a package module
     package_basepath = ENV['DD_INTERNAL_RUBY_INJECTOR_BASEPATH'] || File.expand_path(File.join(File.dirname(__FILE__), '..'))
-    package_gem_home = ENV['DD_INTERNAL_RUBY_INJECTOR_GEM_HOME'] || File.join(package_basepath, RUBY.api_version)
+    package_gem_home = ENV['DD_INTERNAL_RUBY_INJECTOR_GEM_HOME'] || File.join(package_basepath, 'ruby', RUBY.api_version)
```

Comment:
I don't understand this change... e.g. was it wrong before? If so, why did we not see it?

Reply:
Wait, but not all ENVs are settings for testing, are they? Some are actually used for communicating between forks... in production?

I wish the names of the env vars more clearly reflected this -- the ones used for testing would be nice to have `TEST` somewhere in there?

Reply:
Tracked: https://github.com/DataDog/datadog-injector-rb/issues/41